src/plugins/intel_cpu/src/nodes/kernels/scaled_attn/executor_pa.cpp

```diff
@@ -867,6 +870,24 @@ struct MHAHelper {
         _wv_scratch_b.resize<DATA_TYPE>({batch, kv_len_in_blocks, _Hk, _block_size * rnd_up(_S, _block_size)});
     }
 
+    void init_score_buffers(const PlainTensor& past_lens, const PlainTensor& subsequence_begins) {
+        auto seq_cout = static_cast<int32_t>(past_lens.m_dims[0]);
+        _score_offsets_aligned.resize<int32_t>({past_lens.m_dims[0]});
+        _score_offsets.resize<int32_t>({past_lens.m_dims[0]});
+        int32_t total_kv_len_aligned = 0;
+        int32_t total_kv_len = 0;
+        for (int32_t i = 0; i < seq_cout; i++) {
+            auto q_len = subsequence_begins.ptr<int32_t>()[i + 1] - subsequence_begins.ptr<int32_t>()[i];
+            auto kv_len = past_lens.ptr<int32_t>()[i] + q_len;
+            _score_offsets_aligned.ptr<int32_t>()[i] = total_kv_len_aligned;
+            _score_offsets.ptr<int32_t>()[i] = total_kv_len;
+            total_kv_len_aligned += rnd_up(kv_len, 16);
```

Comment:
Because multiple threads will write to the continuous memory(the length is kv_len), to avoid false sharing we'd better to align them to 16=cache size/sizeof(float). Comment is added to the code.
Nestedtensor is a good idea, if this kind of requirement is generic, we may consider to add the support.